var/spack/repos/builtin/packages/connect-proxy/package.py

```diff
@@ -13,7 +13,7 @@ class ConnectProxy(MakefilePackage):
     homepage = "https://bitbucket.org/gotoh/connect"
     url      = "https://bitbucket.org/gotoh/connect/get/1.105.tar.bz2"
 
-    version('1.105', sha256='07366026b1f81044ecd8da9b5b5b51321327ecdf6ba23576271a311bbd69d403')
+    version('1.105', '07366026b1f81044ecd8da9b5b5b51321327ecdf6ba23576271a311bbd69d403')
```

Comment:
What was the issue here?

Reply:
This is actually a mistake (and I’ll fix after I wake up). I tried changing the package to a new repo and the build failed so I reverted back to the original recipe, and missed this hash detail. So safe to ignore, will fix.

Reply:
okay this should be fixed.